tests/e2e/e2e_test.go

```diff
@@ -37,7 +38,9 @@ func runCmd(cmdS string) string {
 
 	if err != nil {
 		fmt.Println(fmt.Sprint(err) + ": " + stderr.String())
-		os.Exit(1)
+		if len(exitStatus) > 0 {
```

Comment:
why are we checking the length of `exitStatus`?

Reply:
I thought making `runCmd` variadic would be good way to deal with exit when there are errors. For most operations, we wouldn't want failing operations exiting test run. With this current, I can specify a non-zero exit code in case I case the test run to error out when a certain operation fails:

`runCmd(<cmd_to_be_run>, 1)` would fail the test run with os.Exit(1) in case the operation fails. If exit code isn't specified, i.e.  `runCmd(<cmd_to_be_run>`), even if the command fails, it would move on to the next test/operation. 

Let me know if this makes sense.

Reply:
@kadel I have reverted the `runCmd` changes. Will address that issue in another PR with related changes.